openedx/features/caliper_tracking/base_transformer.py

```diff
@@ -54,5 +48,23 @@ def add_actor_info(self):
             'id': user_profile_link,
         })
 
+    def __add_extensions(self):
+        self.caliper_event['extensions'] = dict()
```

Comment:
Try to use `{}` instead of `dict()`. It's easier to understand

Reply:
Just for consistency.

Reply:
Updated other occurances as well.